tests/dataset/test_dataset.py

```diff
@@ -12,53 +13,109 @@
 def test_annotations():
 
     bbox = [100, 100, 100, 100]
-    mask = [110, 110, 130, 130, 110, 130]
+    segmentation = [110, 110, 130, 130, 110, 130]
     keypoints = [0, 0, 0, 130, 130, 1, 110, 130, 2]
 
-    ann = A.classification(ann_id=1, img_id=1, cat_id=1)
+    # annotation test
+    ann = A.classification(annotation_id=1, image_id=1, category_id=1)
```

Comment:
So, as an example, your plan is switching this short name `ann` to full name `annotation`. Am i understanding right?

Reply:
Naming rule is our convention.
This code is a usecase, so it depends on users.